emails/tests/models_tests.py

```diff
@@ -511,6 +511,24 @@ def test_display_name_does_not_exist(self):
         profile = Profile.objects.get(user=social_account.user)
         assert profile.display_name == None
 
+    @override_settings(PREMIUM_RELEASE_DATE=datetime.fromisoformat('2021-10-27 17:00:00+00:00'))
+    def test_user_joined_before_premium_release_returns_True(self):
+        user = baker.make(
+            User,
+            date_joined=datetime.now(timezone.utc)
```

Comment:
```suggestion
            date_joined=datetime.fromisoformat('2021-10-18 17:00:00+00:00')
```
or this test will start failing on premium release day.